src/cron/pullCronIdentifiersToCsv.ts

```diff
@@ -71,19 +72,18 @@ export async function pullCronIdentifiersToCsv({
   );
 
   // Write out to CSV
-  writeCsv(
-    file,
-    identifiers.map(({ attributes, ...identifier }) => ({
-      ...identifier,
-      ...attributes.reduce(
-        (acc, val) =>
-          Object.assign(acc, {
-            [val.key]: val.values.join(','),
-          }),
-        {},
-      ),
-    })),
-  );
+  const data = identifiers.map(({ attributes, ...identifier }) => ({
+    ...identifier,
+    ...attributes.reduce(
+      (acc, val) =>
+        Object.assign(acc, {
+          [val.key]: val.values.join(','),
+        }),
+      {},
+    ),
+  }));
+  const headers = uniq(data.map((d) => Object.keys(d)).flat());
```

Comment:
I'm a little unclear on what would happen if there was a duplicate. Say `data` contained something like 
```javascript
[
  {
    id: values1,
    name: values2,
  },
  {
    id: values3,
    location: values4,
  },
];
```
then `headers = ['id', 'name', location']` but would the third column then contain `values3` (being the 3rd column) or `values4` (being the location column)?

Reply:
not sure. follow, i want it to be the superset of all columns?